ontopy/excelparser.py

```diff
@@ -304,8 +307,12 @@ def create_ontology_from_pandas(  # pylint:disable=too-many-locals,too-many-bran
                     if force is True:
```

Comment:
We want to inform about missing labels. Create the message here and use it below...

```python
    msg = (
        f"Error in Property assignment for: {concept}. "
        f"Property to be Evaluated: {prop}. "
        f"Error is {exc}."
    )
```